tests/test_prettytable.py

```diff
@@ -684,6 +695,163 @@ def testHtmlOutputFormatted(self):
+        self.x.title = "Title"
+        self.x.hrules = ALL
+        assert (
+            self.x.get_string().strip()
+            == """
+╔═════════════════════════════════════════════════╗
+║                      Title                      ║
+╠═══════════╦══════╦════════════╦═════════════════╣
+║ City name ║ Area ║ Population ║ Annual Rainfall ║
+╠═══════════╬══════╬════════════╬═════════════════╣
+║  Adelaide ║ 1295 ║  1158259   ║      600.5      ║
+╠═══════════╬══════╬════════════╬═════════════════╣
+║  Brisbane ║ 5905 ║  1857594   ║      1146.4     ║
+╠═══════════╬══════╬════════════╬═════════════════╣
+║   Darwin  ║ 112  ║   120900   ║      1714.7     ║
+╠═══════════╬══════╬════════════╬═════════════════╣
+║   Hobart  ║ 1357 ║   205556   ║      619.5      ║
+╠═══════════╬══════╬════════════╬═════════════════╣
+║   Sydney  ║ 2058 ║  4336374   ║      1214.8     ║
+╠═══════════╬══════╬════════════╬═════════════════╣
+║ Melbourne ║ 1566 ║  3806092   ║      646.9      ║
+╠═══════════╬══════╬════════════╬═════════════════╣
+║   Perth   ║ 5386 ║  1554769   ║      869.4      ║
+╚═══════════╩══════╩════════════╩═════════════════╝""".strip()
+        )
+
+    def testVrulesNone(self):
+        self.x.vrules = NONE
+        assert (
+            self.x.get_string().strip()
+            == "═══════════════════════════════════════════════════\n"
+            "  City name   Area   Population   Annual Rainfall  \n"
+            "═══════════════════════════════════════════════════\n"
+            "   Adelaide   1295    1158259          600.5       \n"
+            "   Brisbane   5905    1857594          1146.4      \n"
+            "    Darwin    112      120900          1714.7      \n"
+            "    Hobart    1357     205556          619.5       \n"
+            "    Sydney    2058    4336374          1214.8      \n"
+            "  Melbourne   1566    3806092          646.9       \n"
+            "    Perth     5386    1554769          869.4       \n"
+            "═══════════════════════════════════════════════════".strip()
+        )
+
+    def testVrulesFrameWithTitle(self):
+        self.x.vrules = FRAME
+        self.x.title = "Title"
+        assert (
+            self.x.get_string().strip()
+            == """
+╔═════════════════════════════════════════════════╗
+║                      Title                      ║
+╠═════════════════════════════════════════════════╣
+║ City name   Area   Population   Annual Rainfall ║
+╠═════════════════════════════════════════════════╣
+║  Adelaide   1295    1158259          600.5      ║
+║  Brisbane   5905    1857594          1146.4     ║
+║   Darwin    112      120900          1714.7     ║
+║   Hobart    1357     205556          619.5      ║
+║   Sydney    2058    4336374          1214.8     ║
+║ Melbourne   1566    3806092          646.9      ║
+║   Perth     5386    1554769          869.4      ║
+╚═════════════════════════════════════════════════╝""".strip()
+        )
+
+    def testEmpty(self):
```

Comment:
I think we could remove this last case? We don't test the empty table for other styles, and I guess it's kind of undefined? Or do you think there's value checking this?

Reply:
Agree - done